File: PHP/aaah.php
<?php

/*
 *	Problem: "Aaah!"
 */
  
 fscanf(STDIN, "%s", $jon);
 fscanf(STDIN, "%s", $doc);
 
 if ( strlen($jon) < strlen($doc) ) {
	 echo "no";
 }
 else {
	 echo "go";
 }
 
?>

File: C++/aaah.cpp
/*
 *	Problem: "Aaah!"
 */
 
#include <iostream>
#include <string>

using namespace std;

/*-----main-----*/
int main(void) {
	string jon;
	string doc;
	
	getline(cin, jon);
	getline(cin, doc);
		
	if ( jon.length() < doc.length() )
		cout << "no" << endl;
	else
		cout << "go" << endl;

	return 0;
}
